src/main/java/ch/hearc/todont/models/User.java

```diff
@@ -0,0 +1,123 @@
+package ch.hearc.todont.models;
+
+import java.util.Set;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+    // FIELDS
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(
+        name = "name",
+        length = 16,
+        nullable = false
+    )
+    private String name;
+
+    @Column(
+        name = "email",
+        nullable = false
+    )
+    private String email;
+
+    @Column(
+        name = "password",
+        length = 255,
```

Comment:
Depending on the hash we will use, I'm not sure we need a 255 characters long field here

Reply:
I left it knowing we might change it later on. As long as we don't start deploying before changing this length to an appropriate value, there's no problem. We just have to not forget it.